zebedee-reader/src/test/java/com/github/onsdigital/zebedee/util/mertics/events/SplunkEventTest.java

```diff
@@ -1,6 +1,6 @@
 package com.github.onsdigital.zebedee.util.mertics.events;
 
-import com.github.davidcarboni.restolino.framework.HttpMethod;
+import com.github.davidcarboni.restolino.api.HttpMethod;
```

Comment:
Worth noting that this change is purely due to the class having moved in this commit.
https://github.com/ONSdigital/restolino/commit/bbd03d367f84458c27900e829c4897cb6a040db5